src/services/api-clients/pricing-client.js

```diff
@@ -0,0 +1,17 @@
+import { BaseClient } from "./base-client";
+
+// This client just gets random pricing stuff to help us determine loan eligibility
+export class PricingClient extends BaseClient {
+  constructor() {
+    super()
```

Comment:
You can just not have the constructor() in here since the parent constructor will be called if there isn't one for the derived